dtfs-central-api/package.json

```diff
@@ -46,6 +46,7 @@
     "validator": "^13.11.0"
   },
   "devDependencies": {
+    "@types/jest": "^29.5.7",
```

Comment:
what's this needed for?

Reply:
We have this in most of the projects but not in `dtfs-central-api` and `portal-api` for some reason. Without this some IDEs have trouble recognising jest functions like `beforeAll`. This will also give better auto-complete.